docs/bundles/11-contributing.md

```diff
@@ -0,0 +1,22 @@
+# Contribting to Massdriver Bundles
+
+## A Note on Bundle Philosophy
+Bundles are intended to be tightly use-case scoped, intention-based reusable pieces of IaC
+for use in the Massdriver platform, see [Building the Right Size Bundle](https://docs.massdriver.cloud/bundles/development#building-the-right-sized-bundle).  
+For this reason, major feature additions that broaden the scope of and existing bundle are likely to be rejected by the community.
+
+
+There are several ways to contribute Massdriver bundles:
+- Fix bugs / minor improvements to existing bundles in the massdriver-cloud org
+- Fork an existing bundle if you are trying to make a major feature change or support a different use case with similar technology.
```

Comment:
Wondering if we should revise this a bit for now. I don't know if "forking" should be under "ways to contribute" since they aren't really contributing...they are taking on ownership of their own bundle to service their own use case.